packages/react-components/react-combobox/library/src/contexts/usePromptInputContextValues.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { PromptInputState } from '../PromptInput';
+import { ComboboxBaseContextValues, ComboboxBaseState } from '../utils/ComboboxBase.types';
+
+export function usePromptInputContextValues(
```

Comment:
```
useGenericInputContextValues = (state) => {
  useComboboxInputcontextValue({...state, appearance: 'outline' });
}
```

Reply:
Repeat code and strip deprecated props, basically keep active descendant and necessary input stuff like setOpen

Reply:
This should be in collab